cpp/src/gandiva/lru_cache.h

```diff
@@ -113,9 +196,32 @@ class LruCache {
     lru_list_.erase(i);
   }
 
+  void evictObject() {
```

Comment:
Ditto

Reply:
I checked, and I might be the case, so I capitalized the first letter.